tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource

```diff
@@ -26,10 +27,16 @@ ${MS_TABLE_STATUS_FAILURE}=    //span[contains(@class,"pf-c-icon__content")][con
 *** Keywords ***
 Open Model Serving Home Page
     [Documentation]    Verifies submenu Settings > "Model Serving" is visible and navigates to page
-    Page Should Contain    Model Serving
-    Click Link      Model Serving
-    Wait Until Page Contains    Manage and view the health and performance of your deployed models.    timeout=30
-    Wait Until Page Contains Element    ${MS_HEADER_XP}    timeout=30
+    SeleniumLibrary.Page Should Contain    Model Serving
+    SeleniumLibrary.Click Link      Model Serving
+    Sleep    1s
+    ${loading_error} =    Run Keyword And Return Status    SeleniumLibrary.Page Should Contain    Problem loading model serving page
+    IF    ${loading_error}
+        Sleep    5s
+        SeleniumLibrary.Reload Page
+    END
+    SeleniumLibrary.Wait Until Page Contains    Manage and view the health and performance of your deployed models.    timeout=30
+    SeleniumLibrary.Wait Until Page Contains Element    ${MS_HEADER_XP}    timeout=30
```

Comment:
I think you could replace this with `Wait for RHODS Dashboard to Load`

Reply:
the Wait kw would look for the same element `xpath://h1[text()="${expected_page}"]` vs `    xpath=//h1[text()="Deployed models"]`